index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Signup Form</title>
+    <meta name="description" content="Sign up for an intuit account">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+
+
+    <div id="sign-in">
+        <p>Already have an Intuit account?<br>
+            I forgot my User ID or Password Sign In</p>
+        <button>Sign in</button>
+    </div>
+
+    <header>
+        <img alt="Intuit logo" id="intuit-logo" src="assets/logo-intuit.png">
+
+        <div id="small-logos">
+            <img alt="Mint logo" src="assets/logo-mint.png">
+            <img alt="quickbooks logo" src="assets/logo-quickbooks.png">
+            <img alt="turbotax logo" src="assets/logo-turbotax.png">
+
+        </div>
```

Comment:
- **Intuit products**
You shouldn't use screenshots of the original page as a shortcut to bypass requirements, Please fix them:
- [ ] It shows the Mint logo and text (logo on the left, text on the right) (link to page)
- [ ] It shows the Quickbooks logo and text (logo on the left, text on the right) (link to page)
- [ ] It shows the Turbotax logo and text (logo on the left, text on the right) (link to page)

![image](https://user-images.githubusercontent.com/60184961/93373331-e07a9b80-f85d-11ea-9e53-69ad57793e7f.png)